db/migrate/20210204164407_add_attachment_checksums.rb

```diff
@@ -0,0 +1,8 @@
+class AddAttachmentChecksums < ActiveRecord::Migration[6.0]
```

Comment:
not a blocker - generally better, particularly if deploys are slow, to get migrations in as a separate deploy so long as nil values are considered valid/no-db-contraints. Not doing so could risk conditions where the code is updated just as someone is saving an attachment but migrations have not been applied - Not sure this is possible but we have had instances in the past where we have had errors related to code and migration changes going in together.

Reply:
although i think it is ore the reverse - if db migration gets applied first but code is still old code, which i do not believe will matter in this scenario?!